src/stories/Library/event-header/EventHeader.tsx

```diff
@@ -0,0 +1,46 @@
+import { FC } from "react";
+import { Tag } from "../tag/Tag";
+import { Button } from "../Buttons/button/Button";
+
+type EventHeaderProps = {
+  title: string;
+  date: string;
+};
+
+const EventHeader: FC<EventHeaderProps> = ({ title, date }) => {
+  return (
+    <header className="event-header">
+      <section className="event-header__content">
+        <div className="event-header__extra-padding">
```

Comment:
Please change the name of the `event-header__extra-padding` class.

It is a *very* specific name. If possible I would like to avoid the div entirely. I wondered if we could implement it like a three element grid (image, content and state) where all elements are in one column on mobile and in two colums on desktop. Image in one and content + state in the other.

If that is not possible then I would perfer a better class name.